frontend/src/libs/components/meditation-player/audio-player/audio-player.tsx

```diff
@@ -0,0 +1,81 @@
+import { useCallback, useRef, useState } from '#libs/hooks/hooks.js';
+import { type Meditation } from '#libs/types/types.js';
+
+import { START_TIME, STEP } from '../constants/constants.js';
```

Comment:
give more meaningful names to constants, like TRACK_INCREMENT_INDEX
```suggestion
import { START_TIME, STEP } from '../constants/constants.js';
```